aten/src/ATen/native/Normalization.cpp

```diff
@@ -58,6 +58,10 @@
 #include <ATen/ops/sqrt.h>
 #endif
 
+#ifdef USE_CUDA
+#include <ATen/cuda/CUDAContext.h>
```

Comment:
My understanding is that `Normalization.cpp` shouldn't depend on `CUDAContext.h`; the typical way that this kind of support check is done is via `CUDAHooks.cpp` which can query things like the device properties if CUDA is available---`Normalization.cpp` can call into `CUDAHooks` without needing to depend on anything CUDA-specific. You can see examples of other support-checking functions https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/cuda/detail/CUDAHooks.cpp